packages/react-composites/tests/browser/common/hermeticChatTestHelpers.ts

```diff
@@ -15,7 +15,10 @@ import { dataUiId, perStepLocalTimeout, screenshotOnFailure } from './utils';
  */
 export async function temporarilyShowHiddenChatComposite(page: Page, participant: ChatParticipant): Promise<void> {
   await withHiddenChatCompositeInForeground(page, participant, async () => {
-    return;
+    // wait for messages to have loaded
+    await waitForSelector(page, dataUiId('chat-composite-message'));
+    // sleep for 100ms to give time for the messagethread hook to send the read receipt
```

Comment:
Maybe we need a hidden element in html to notify that "I have successfully sent the read receipt at what time point"

Reply:
Yeah :/ Would be nice to use the adapter and not composites for these actually and use adapter interface to send the read receipt